src/shared-components/dropdown/test.tsx

```diff
@@ -58,30 +59,30 @@ describe('<MobileDropdown />', () => {
 
 describe('<DesktopDropdown />', () => {
   it('renders the current option text', () => {
-    const wrapper = shallow(
+    const wrapper = mount(
       <DesktopDropdown
         options={options}
         currentOption={{ value: 'test1', label: 'Test1' }}
-      />
+      />,
     );
 
     expect(
       wrapper
         .find('#select-input-box')
         .text()
-        .includes('Test1')
+        .includes('Test1'),
     ).toEqual(true);
   });
 
   describe('onSelectClick callback', () => {
     it('should be invoked onClick', () => {
       const spy = jest.fn();
-      const wrapper = shallow(
+      const wrapper = mount(
```

Comment:
These tests were failing when using `shallow` - I believe the issue is that the `hook` was still firing, but it seemed like it was coming from the render of the `class` component, which violates the laws of Hooks. Using `mount` does a more full render of the tree and seemed to fix the issue.